assessment/serializers.py

```diff
@@ -55,11 +61,31 @@ class Meta:
             'questions',
             'is_admin',
             'in_users',
+            'user_survey_status',
             'name',
             'description',
-
         )
 
+    def get_user_survey_status(self, obj):
+        """
+        When an authenticated user request survey details,
+        it should include if the survey has been started, completed, etc.
+        """
+        try:
+            user = self.context.get('request').user
+        except Exception:
+            # raise serializers.ValidationError('Could not access request.user')
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3AS125)